src/extensions/default/SVGCodeHints/SVGTags.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
This format looks great!

Since this data has both *tags* and *attributes*, I think the term "groups" is a little vague. What about using "attributeGroups" ?